### ISSUE TITLE
Uncaught TypeError: Cannot read property 'listExists' of undefined

### ISSUE THREAD

**Issue by Kosai106** [imdone/imdone-atom#55]
1. Add TODO to a list
2. Change the name of said list
3. It also appears that said error caused the TODO's on the list to get removed, and adding new ones to it, removes them as well.

**Atom Version**: 1.2.4
**System**: Microsoft Windows 10 Pro
**Thrown From**: [imdone-atom](https://github.com/imdone/imdone-atom) package, v1.3.4
### Stack Trace

Uncaught TypeError: Cannot read property 'listExists' of undefined

```
At C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\repository.js:322

TypeError: Cannot read property 'listExists' of undefined
    at Repository.listExists (C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\repository.js:322:26)
    at File.taskFoundListener (C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\repository.js:227:15)
    at emitOne (events.js:77:13)
    at File.emit (events.js:169:7)
    at File.extractCodeStyleTasks (C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\file.js:336:12)
    at File.extractTasksInCodeFile (C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\file.js:384:10)
    at File.extractTasks (C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\file.js:530:10)
    at extract (C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\repository.js:527:10)
    at Repository.extractTasks (C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\repository.js:538:10)
    at C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\repository.js:617:12
    at C:\Users\WebDev\.atom\packages\imdone-atom\node_modules\imdone-core\lib\mixins\repo-fs-store.js:366:9
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:380:3)
```
### Commands

```
  2x -0:49.7.0 imdone-atom:tasks (atom-pane.pane.active)
  2x -0:06.2.0 core:move-left (atom-text-editor.editor.mini.is-focused)
     -0:05.8.0 core:backspace (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {},
  "imdone-atom": {
    "openIn": {}
  }
}
```
### Installed Packages

``` coffee
# User
atom-bootstrap4, v1.2.13
autoclose-html, v0.19.0
battery-status, v0.10.3
highlight-selected, v0.11.1
html2haml, v0.10.0
imdone-atom, v1.3.4
imdone-atom-github, v0.1.7
language-haml, v0.23.12
linter, v1.11.3
linter-csslint, v1.1.0
linter-haml, v1.0.0
linter-sass-lint, v0.4.1
minimap, v4.16.2
pigments, v0.19.1
spotify-remote, v0.5.2

# Dev
No dev packages
```

**Comment by piascikj** [comment]
Are you still getting this error?

**Comment by Kosai106** [comment]
@piascikj No luckily not, thanks. I must say I really like this plugin, so it's great to have it functional again.

Another slightly annoying thing I've noticed however, which you probably can't do anything about: If I add 
```
// TODO: Something
```
in a HAML file - imdone won't pick it up it seems.
My solution so far has been just adding the TODO's in an otherwise empty HTML file, or within my SCSS, but still, I figured I'd let you know.

Cheers.
